### PR TITLE
:tada: Add workspace read-only setting

### PR DESCRIPTION
**Pull request by pabloalba**
Implements https://tree.taiga.io/project/penpot/task/4514

### How to test
* Call in the console to debug.set_workspace_read_only(true) or debug.set_workspace_read_only(false) to activate/deactivate read only

### CODE REVIEW

File: frontend/src/app/main/data/workspace/state_helpers.cljs [hunk 1]
```diff
@@ -146,3 +146,9 @@
   (let [{:keys [x y width height]} (get-in state [:workspace-local :vbox])]
     (gpt/point (+ x (/ width 2)) (+ y (/ height 2)))))
 
+
+(defn lookup-workspace-read-only?
```
Comment (niwinz) [inline frontend/src/app/main/data/workspace/state_helpers.cljs]: Looks unused

File: frontend/src/app/main/ui/workspace/header.cljs [hunk 2]
```diff
@@ -464,7 +468,8 @@
                 :project project
                 :file file
                 :team-id team-id
-                :page-id page-id}]]
+                :page-id page-id
+                :workspace-read-only? workspace-read-only?}]]
```
Comment (niwinz) [inline frontend/src/app/main/ui/workspace/header.cljs]: You don't need to pass this all down having the react context set. Just use the use-ctx on the component if you need to know the value

Comment (pabloalba) [reply frontend/src/app/main/ui/workspace/header.cljs]: Fixed!

File: frontend/src/app/main/data/workspace/state_helpers.cljs (hunk 1 above)
Comment (pabloalba) [reply frontend/src/app/main/data/workspace/state_helpers.cljs]: Removed

**Pull request merged**